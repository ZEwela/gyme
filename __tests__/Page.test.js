import React from "react";
import renderer from "react-test-renderer";

import Page from "../app/index.js";

describe("<Page />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Page />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
