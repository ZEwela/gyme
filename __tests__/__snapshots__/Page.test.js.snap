// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 24,
    }
  }
>
  <Text
    accessibilityRole="link"
    href="/workouts"
    onPress={[Function]}
    style={
      {
        "borderWidth": 0.5,
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 64,
          "fontWeight": "bold",
        }
      }
    >
      Pick
      <Text />
       workout
    </Text>
  </Text>
</View>
`;
